change loadable default type (#1291)
seek-oss/sku: .changeset/slimy-bats-burn.md

---
'@sku-lib/vite': patch
---

> `loadable`: The factory type now has an optional `default` property. Not all components loaded will have a `default` export.

Reviewer: `nit`: just making it clearer that the default prop has changed and not added 

```suggestion
`loadable`: The factory type now sets `default` as option. Not all components loaded will have a `default` export.
```
